package.json

```diff
@@ -42,71 +42,64 @@
     "tests"
   ],
   "dependencies": {
-    "@fastify/cors": "^8.3.0",
-    "@fastify/static": "^6.10.2",
-    "@types/ws": "^8.5.5",
-    "@vscode/test-electron": "^2.3.4",
-    "@wdio/logger": "^8.11.0",
+    "@fastify/cors": "^8.4.1",
+    "@fastify/static": "^6.12.0",
+    "@types/ws": "^8.5.10",
+    "@vscode/test-electron": "^2.3.8",
+    "@wdio/logger": "^8.16.17",
     "clipboardy": "^3.0.0",
     "decamelize": "6.0.0",
     "download": "^8.0.0",
-    "fastify": "^4.21.0",
+    "fastify": "^4.24.3",
     "get-port": "7.0.0",
     "slash": "^5.1.0",
     "tmp-promise": "^3.0.3",
-    "undici": "^5.23.0",
+    "undici": "^5.28.1",
     "vscode-uri": "^3.0.8",
-    "wdio-chromedriver-service": "^8.1.1",
-    "ws": "^8.13.0",
+    "ws": "^8.14.2",
     "yargs-parser": "^21.1.1"
   },
   "peerDependencies": {
-    "chromedriver": "latest",
     "webdriverio": "^8.0.0"
```

Comment:
If I understand the blog post correctly, this will need to explicitly be ^8.14.0, right?